File: src/src/example_package/Example_package.java
import common_msgs.Messages.*;

public class Example_package {
    public void run() {
        while (true) {
            // TODO: Implement node logic
            System.out.println("Running example_package");
            try {
                Thread.sleep(1000);
            } catch (InterruptedException e) {
                e.printStackTrace();
            }
        }
    }

    public static void main(String[] args) {
        Example_package node = new Example_package();
        node.run();
    }
}

File: src/src/example_package/example_package.py
from common_msgs import *
import time

class Example_package:
    def run(self):
        while True:
            # TODO: Implement node logic
            print(f"Running example_package")
            time.sleep(1)

if __name__ == "__main__":
    node = Example_package()
    node.run()
